configs/autopush_endpoint.ini

```diff
@@ -24,3 +24,7 @@ port = 8082
 ;
 ; GCM requires a API Auth key. It is STRONGLY recommended you use one specific to this app.
 ; gcm_apikey = <API_KEY>
+;
+; AuthKey are the keys to use for Bearer Auth tokens. It uses the same
+; autokey generator as the crypto_key argument, and sorted [newest, oldest]
+; auth_key = [HJVPy4ZwF4Yz_JdvXTL8hRcwIhv742vC60Tg5Ycrvw8=]
```

Comment:
I wonder if this might break puppet? In essence, we're specifying invalid JSON here, since a list is created as [key, key]